### PR TITLE
*: ctl v3 tests with multi expects

### PR DESCRIPTION
**Pull request by gyuho**
Resolves TODO with the new expect package.

/cc @heyitsanthony @xiang90

### CODE REVIEW

File: pkg/expect/expect.go [hunk 1]
```diff
@@ -94,6 +94,19 @@ func (ep *ExpectProcess) Expect(s string) (string, error) {
 	return "", ep.err
 }
 
+// Expects expects multiple lines.
+func (ep *ExpectProcess) Expects(txts ...string) ([]string, error) {
+	var lines []string
+	for _, txt := range txts {
+		s, err := ep.Expect(txt)
+		if err != nil {
+			return nil, err
```
Comment (heyitsanthony) [inline pkg/expect/expect.go]: return partial result?

File: e2e/etcd_test.go [hunk 2]
```diff
@@ -434,6 +434,19 @@ func spawnWithExpect(args []string, expected string) error {
 	return perr
 }
 
+func spawnWithExpects(args []string, exps ...string) error {
```
Comment (heyitsanthony) [inline e2e/etcd_test.go]: spawnWithExpect should call this instead of having a separate implementation

File: pkg/expect/expect.go (hunk 1 above)
Comment (gyuho) [inline pkg/expect/expect.go]: Fixed. Thanks!

File: e2e/etcd_test.go (hunk 2 above)
Comment (gyuho) [inline e2e/etcd_test.go]: Ok will fix.

File: pkg/expect/expect_test.go [hunk 3]
```diff
@@ -40,3 +38,20 @@ func TestEcho(t *testing.T) {
 		t.Fatalf("expected error on closed expect process")
 	}
 }
+
+func TestExpects(t *testing.T) {
+	ep, err := NewExpect("ping", "-c3", "127.0.0.1")
```
Comment (heyitsanthony) [inline pkg/expect/expect_test.go]: `seq` instead of `ping`?

**Comment by heyitsanthony** [comment]
lgtm following fix ups

**Comment by gyuho** [comment]
thanks. just switched it to `seq`. The special characters are not escaped in that command though.

**Comment by heyitsanthony** [comment]
On the other hand-- could avoid the test all together by rolling all the Expects logic into spawnWithExpects and not touch the expect package...

**Comment by gyuho** [comment]
yeah that's simpler. let me try that.

File: e2e/etcd_test.go [hunk 4]
```diff
@@ -422,11 +422,20 @@ func spawnCmd(args []string) (*expect.ExpectProcess, error) {
 }
 
 func spawnWithExpect(args []string, expected string) error {
+	return spawnWithExpects(args, []string{expected}...)
+}
+
```
Comment (gyuho) [inline e2e/etcd_test.go]: @heyitsanthony PTAL. Thanks!

**Comment by heyitsanthony** [comment]
lgtm

**Pull request merged**